mkdocs.yml

```diff
@@ -214,8 +215,8 @@ nav:
             - Modelo 347: user-guide/etendo-classic/optional-features/bundles/spain-localization/modelo-347.md
             - Modelo 349: user-guide/etendo-classic/optional-features/bundles/spain-localization/modelo-349.md
             - Modelo 390: user-guide/etendo-classic/optional-features/bundles/spain-localization/modelo-390.md
-            - Overview: user-guide/etendo-classic/optional-features/bundles/spain-localization/overview.md
             - SII para IVA: user-guide/etendo-classic/optional-features/bundles/spain-localization/sii-para-iva.md
+            - TicketBai: user-guide/etendo-classic/optional-features/bundles/spain-localization/ticketbai.md
```

Comment:
If you want "TicketBai and Batuz" to be shown in the title and navigation, you have to change it here. 
![image](https://github.com/user-attachments/assets/f9be3b3a-44f4-4048-bac2-9d84f6834b5d)